docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -49,22 +114,147 @@ GET /_search
 --------------------------------------------------
 // CONSOLE
 
-The query `path` points to the nested object path, and the `query`
-includes the query that will run on the nested docs matching the
-direct path, and joining with the root parent docs. Note that any
-fields referenced inside the query must use the complete path (fully
-qualified).
```

Comment:
The original is a bit of a word salad, but I feel like an explanation of the query path and fully-qualified field paths is still needed.